deps/graph-parser/src/logseq/graph_parser/extract.cljc

```diff
@@ -16,7 +16,8 @@
             [logseq.graph-parser.config :as gp-config]
             #?(:org.babashka/nbb [logseq.graph-parser.log :as log]
                :default [lambdaisland.glogi :as log])
-            [logseq.graph-parser.whiteboard :as gp-whiteboard]))
+            [logseq.graph-parser.whiteboard :as gp-whiteboard]
+            [medley.core :as medley]))
```

Comment:
Adding new dependencies breaks existing usage of nbb-logseq for the graph-parser without some coordination with nbb-logseq. I put this in the [graph-parser readme](https://github.com/logseq/logseq/tree/master/deps/graph-parser#managing-dependencies) but happy to put the notice somewhere else too if it's helpful. Adding medley to nbb-logseq is probably not worth the effort for just one fn. For now could you just copy [the fn](https://github.com/weavejester/medley/blob/d1e00337cf6c0843fb6547aadf9ad78d981bfae5/src/medley/core.cljc#L22-L38) into graph-parser.util? If we end up using more medley fns, I can look into adding medley to nbb-logseq

Reply:
Cool, I just copied `dissoc-in` into graph-parser.util.